docs/release/trg-5/trg-5-05.md

```diff
@@ -47,3 +49,55 @@ This might cause unexpected effects, as the Helm release could change without bu
 security reasons you shall not use `latest` tag.
 
 :::
+
+## Ingress
+
+- Ingress **must** be disabled by default
+- Hostname **must** not be set by default
+
+An example section for ingress resources:
+
+```yaml
+ingress:
+  enabled: false
+  # className: "nginx"
+  ## Optional annotations when using the nginx ingress class
+  #annotations:
+  #  nginx.ingress.kubernetes.io/rewrite-target: "/$1"
+  #  nginx.ingress.kubernetes.io/use-regex: "true"
+  #  nginx.ingress.kubernetes.io/enable-cors: "true"
+  #  nginx.ingress.kubernetes.io/cors-allow-origin: "https://*.example.org"
+  tls:
+    - secretName: ""
+      hosts:
+        - ""
+  hosts:
+    - host: ""
+      paths:
+        - path: "/(.*)"
+          pathType: "Prefix"
+          backend:
+            service: "app"
+            port: 8080
+```
+
+## Dependencies
+
+An example section for Postgresql dependency configuration:
+
+```yaml
+postgresql:
+  ## PostgreSQL chart configuration
+  enabled: true
+  ## Dependencies should be prefixed with the `nameOverride` and/or `fullnameOverride` properties
```

Comment:
I'd refrain from recommending fullnameOverride(s) as especially in the case of umbrella charts unique labels (name) for dependencies are important and a fullnameOverride will lead to non-unique labels (at least if there are >1 fullnameOverride(s) in the umbrella).

Reply:
Point taken, I've removed the fullNameOverride from the comment.